plugins/polio/admin.py

```diff
@@ -130,11 +131,16 @@ class VaccineAuthorizationsAdmin(admin.ModelAdmin):
     raw_id_fields = ("country",)
 
 
+class ReasonForDelayAdmin(admin.ModelAdmin):
```

Comment:
```suggestion
@admin.register(ReasonForDelay)
class ReasonForDelayAdmin(admin.ModelAdmin):
```

[This decorator](https://docs.djangoproject.com/en/4.2/ref/contrib/admin/#the-register-decorator) allows to keep things closer instead of putting everything at the end of the file:

~`admin.site.register(ReasonForDelay, ReasonForDelayAdmin)`~